src/renderer/components/swap/Swap.utils.ts

```diff
@@ -257,3 +257,9 @@ export const minBalanceToSwap = (swapFees: Pick<SwapFees, 'inFee'>): BaseAmount
   // Over-estimate balance by 50%
   return feeToCover.times(1.5)
 }
+
+export const maxAmountToSwapMax1e8 = (assetAmount: BaseAmount, inFeeAmount: BaseAmount): BaseAmount => {
+  const estimatedFee = to1e8BaseAmount(inFeeAmount.times(1.5))
```

Comment:
`to1e8BaseAmount` -> `max1e8BaseAmount` (because passed `assetAmount` is `max1e8` based)